api/src/main/java/com/epam/pipeline/entity/templates/FolderTemplate.java

```diff
@@ -33,6 +33,7 @@
 @Builder
 public class FolderTemplate {
     private String name;
+    private Long parentId;
```

Comment:
I would suggest to store home folder parent id in `SystemPreferences`

Reply:
Done